packages/salesforce/src/Adaptor.js

```diff
@@ -298,21 +320,19 @@ const defaultOptions = {
  * `bulkQuery()` uses {@link https://sforce.co/4azgczz Bulk API v.2.0 Query} which is available in API version 47.0 and later.
  * This API is subject to {@link https://sforce.co/4b6kn6z rate limits}.
  * @public
- * @example
- * <caption>The results will be available on `state.data`</caption>
- * bulkQuery(state=> `SELECT Id FROM Patient__c WHERE Health_ID__c = '${state.data.field1}'`);
- * @example
- * bulkQuery(
- *   (state) =>
- *     `SELECT Id FROM Patient__c WHERE Health_ID__c = '${state.data.field1}'`,
- *   { pollTimeout: 10000, pollInterval: 6000 }
- * );
+ * @example <caption>Perform a bulk query on Patient records</caption>
+ * bulkQuery('SELECT Id, Name FROM Patient__c');
```

Comment:
These examples don't use any dynamic state. We should make sure there's at least one example which uses state (or lazy state)